scripts/defaults/jaseci.yaml

```diff
@@ -61,11 +61,11 @@ rules:
       "validatingwebhookconfigurations",
       "volumeattachments"
     ]
-    verbs: ["get", "watch", "list", "update", "patch", "create"]
+    verbs: ["get", "watch", "list", "update", "patch", "create", "delete"]
```

Comment:
Oh....what is the difference between this yaml and the one in `scripts/jaseci.yaml`?